src/gui/dialogs/export_project.ui

```diff
@@ -10,6 +10,12 @@
     <height>412</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>519</width>
```

Comment:
Ok, no need to change them then, it's just that I found them odd at a first glance ;)